_data/simple-icons.json

```diff
@@ -75,6 +75,11 @@
             "hex": "FF5A5F",
             "source": "https://www.airbnb.com"
         },
+        {
+            "title": "Go Language",
```

Comment:
Seems like they prefer to use simply "Go" (see e.g. [their site](https://golang.org/))

Reply:
My reasoning for using "Go Language" over "Go" is that they use "The Go Programming Language" in both the header and page title (`<head><title>The Go Programming Language</title></head>`). But I'm okay with just "Go" as well

Reply:
They add extra qualifiers a lot ("golang", "The Go Programming Language"), but also seem to use the plain name whenever the context is established ("Try Go", "Go is an open source [...]", "Dowwnload Go"). Seems that "Go" is the official name, but they realize that it is slightly ambiguous.